.devops/deploy-pipelines.yml

```
@@ -0,0 +1,69 @@
+# Automatically triggered on PR
+# Only triggers on branches
+pr:
+  autoCancel: true
+  branches:
+    exclude:
+     - '*'
+
+trigger:
+  branches:
+    exclude:
+      - master
+  paths:
+    include:
+      - 'src/*'
+      - '.devops/*'
+
+parameters:
+  - name: 'SUBSCRIPTION'
+    displayName: 'Azure subscription hosting the infrastructure built with terraform'
+    type: string
+    default: DEV-CSTAR-SERVICE-CONN
+    values:
+      - PROD-CSTAR-SERVICE-CONN
+      - UAT-CSTAR-SERVICE-CONN
+      - DEV-CSTAR-SERVICE-CONN
+pool:
+  vmImage: 'ubuntu-latest'
```

```suggestion
  vmImage: 'ubuntu-20.04'
```
pin vm image version